setup.py

```diff
@@ -24,7 +23,7 @@ def run_tests(self):
 
 setup(
     name="PyHopsHive",
-    version=pyhive.__version__,
+    version="0.6.4.1",
```

Comment:
Can you please bump the version here: https://github.com/logicalclocks/PyHive/blob/02531c16529c546146e247cc0bfb8a8a7e47a5cc/pyhive/__init__.py#L3 - So that we don't end up with multiple version numbers in the project?

Reply:
Ah right! fixed :)